main/infrastructure-tests/__test__/appstream-egress-enabled/route-tables.test.js

```diff
@@ -0,0 +1,76 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const AWS = require('aws-sdk');
+const setupAws = require('../../support/setupAws');
+const { getCFStackResources, getStackResourcesByType } = require('../../support/utilities');
+
+describe('Route tables', () => {
+  beforeAll(async () => {
+    await setupAws();
+  });
+
+  it('should not point to any internet gateways', async () => {
+    const stackResources = await getCFStackResources();
+    const ec2 = new AWS.EC2();
+
+    // Grab Route tables created for AppStream VPC
+    const vpcId = await getStackResourcesByType('VPC', stackResources);
+    const routeTablesForDefaultVpcResponse = await ec2
```

Comment:
nit: can we rename this to AppStreamVPC rather than DefaultVPC. Default VPC in AWS has a different meaning.

Reply:
updated